Doc/whatsnew/3.11.rst

```diff
@@ -193,9 +193,9 @@ Other CPython Implementation Changes
   support :class:`typing.SupportsComplex` and :class:`typing.SupportsBytes` protocols.
   (Contributed by Mark Dickinson and Dong-hee Na in :issue:`24234`.)
 
-* ``siphash13`` is added as a new internal hashing algorithms. It's has similar security
+* ``siphash13`` is added as a new internal hashing algorithms. It has similar security
   properties as ``siphash24`` but it is slightly faster for long inputs. ``str``, ``bytes``,
-  and some other types now use it as default algorithm for ``hash()``. :pep:`552`
+  and some other types now use it as the default algorithm for ``hash()``. :pep:`552`
```

Comment:
IMO there wasn’t a problem to fix here.

Thanks for catching the other typos!

Reply:
Yes, please revert this unless you disagree with me!

```suggestion
  and some other types now use it as default algorithm for ``hash()``. :pep:`552`
```

Reply:
I think it needs the definite article "the". The sentence reads less clearly without it. If I have the option to keep the addition then I will, but if it's a blocker for accepting the patch then I'll remove it.

100% agree on converting the `hash` reference to a link. I'll get that done.

Reply:
But the empty article is a valid article in English, and appropriate here!  «Some types use it as default algorithm» has no mistake.

Reply:
I've removed the definite article. Thanks for your feedback and work on Python! 👍